Data/ni.yaml

```diff
@@ -0,0 +1,47 @@
+meta:
+  name: nickel
+  formula: Ni
+  last_updated: 7272181873150
+  description:
+  comment:
+  references:
+    '@online{wikipedia,
+      author = "Wikipedia",
+      title = "Nickel",
+      url = "https://de.wikipedia.org/wiki/Nickel"
+      }
+
+    @online{ri,
+      author = "refractiveindex.info",
+      title = "Nickel",
+      url = "https://refractiveindex.info/?shelf=main&book=Ni&page=Johnson"
+      }'
+
+data:
+  thermal_conductivity:
+    wikipedia:
+      value: 91
+      units: [W/(mK)]
+  molar_volume:
+    wikipedia:
+      value: 6.59e-06
+      units: [m^3/mol]
+  boiling_point:
+    wikipedia:
+      value: 3003.0
+      units: [K]
+  melting_point:
+    wikipedia:
+      value: 1728.0
+      units: [K]
+  refractive_index:
+    ri:
```

Comment:
why do you have this additional layer `ri`?
one option would be to name this layer as the argument, so `wavelength`
I actually like this idea best, because in the `data` section we also need to parse the names of the actual properties, which we do not know in advance, so the same can apply for the arguments of a property

Reply:
This is the citation key, right, maybe bad choice here, as it is short. might want to change it to ri_info, as this a very common source.

Reply:
ah okay, stupid me.  maybe it is even better to use the original source provided bei ref_index.org, but anyway. One could also mark the citation key with an `@`

Reply:
Yes, one could, but would need quotation marks again, see comment below.